components/script/webdriver_handlers.rs

```diff
@@ -168,6 +169,16 @@ unsafe fn object_has_to_json_property(
     }
 }
 
+#[allow(unsafe_code)]
+/// <https://w3c.github.io/webdriver/#dfn-collection>
```

Comment:
Linking to https://w3c.github.io/webdriver/#dfn-collection here seems a bit weird.
Maybe link to https://tc39.es/ecma262/#sec-arguments-exotic-objects instead, and move the https://w3c.github.io/webdriver/#dfn-collection link to where you use `is_array_like(cx, val) || is_arguments_object(cx, val)`

Reply:
I understand the intent here, but the actual implementation is unrelated to the ecmascript link, and there's no way to provide a more specfific link to the first bullet point of the dfn-collection link. I could add text like "bullet 1" to the doc comment, if that would help.